CHANGELOG.md

```diff
@@ -8,6 +8,16 @@
 
 ### Fixed
 
+<a id="1_46_4"></a>
+
+## [1.46.4] - 2024-08-16
+
+### Changed
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.142.6`
+  - Still try to create "INBOX.DeltaChat" if couldn't create "DeltaChat" #5870
+  - Increase timeout for QR generation to 60s #5882
+  - Fix default to strict TLS checks if not configured #5888
```

Comment:
Shouldn't we more explicitly ask people to change their settings if their account was created on version 1.46.3?

Reply:
These changes were not made in dc desktop but in core and are mentioned here only for convenience...

Reply:
Maybe they should rerun the configuration because "fix: default to strict TLS checks if not configured" might affect which IMAP/SMTP servers are chosen during the configuration. But it doesn't look critical, the new default should apply anyway. CC @link2xt